include/cantera/kinetics/RxnRates.h

```diff
@@ -91,6 +91,113 @@ class Arrhenius
     doublereal m_logA, m_b, m_E, m_A;
 };
 
+//! Blowers Masel reaction rate type depends on enthalpy 
+/**
+ * The Blowers Masel approximation is written by Paul Blowers,
+ * Rich Masel(DOI: https://doi.org/10.1002/aic.690461015) to 
+ * adjust the activation energy based on enthalpy change of a reaction:
+ * 
+ *   \f{eqnarray*}{
+ *        Ea &=& 0\; \text{if }\Delta H < -4E_0 \\ 
+ *        Ea &=& \Delta H\; \text{if }\Delta H > 4E_0 \\
+ *        Ea &=& \frac{(w + \Delta H / 2)(V_P - 2w + 
```

Comment:
```suggestion
 *        E_a &=& 0\; \text{if }\Delta H < -4E_0 \\ 
 *        E_a &=& \Delta H\; \text{if }\Delta H > 4E_0 \\
 *        E_a &=& \frac{(w + \Delta H / 2)(V_P - 2w + 
```
Also, please look for and eliminate the trailing whitespace in this docstring.